### PR TITLE
feat(dapps) implement send transaction support for wallet connect

### PR DESCRIPTION
**Pull request by stefandunca**
### Closes: #15126

- [x] rebased on top of https://github.com/status-im/status-desktop/pull/15193

Also added Fees section

https://github.com/status-im/status-desktop/assets/47554641/abd8fedf-58b9-486e-b8ce-36ed17de3dc7

### CODE REVIEW

File: ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml [hunk 1]
```diff
@@ -153,8 +155,14 @@ ConnectedDappsButton {
...
                 target: root.wcService.requestHandler
 
-                function onMaxFeesUpdated(maxFees, symbol) {
+                function onMaxFeesUpdated(maxFees, maxFeesWei, haveEnoughFunds, symbol) {
                     maxFeesText = `${maxFees.toFixed(2)} ${symbol}`
+                    var ethStr = "?"
+                    if (globalUtils) {
+                        ethStr = globalUtils.wei2Eth(maxFeesWei, 9)
+                    }
+                    maxFeesEthText = `${ethStr} ETH`
```
Comment (caybro) [inline ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml:164]: Anything against using the standard `LocaleUtils.formatFoo` here?

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml (hunk 1 above)
Comment (stefandunca) [reply ui/app/AppLayouts/Wallet/panels/DAppsWorkflow.qml:164]: Not sure what you mean. I found only the following function in `LocaleUtils`: `formatDate`, `formatTime`, `formatDateTime` and `formatRelativeTimestamp`.
This code is temporary to have a placeholder until the implementation is finalized for the data source in here [#15190](https://github.com/status-im/status-desktop/issues/15190), [#15191](https://github.com/status-im/status-desktop/issues/15191)

**Comment by kounkou** [comment]
Excuse my ignorance, 

Could you explain how the flow works with the updated files please?

when you click on `eth_sendTransaction` button here in browser :

Screenshot from 2024-06-27 17-19-26

Is the `signTransaction` triggered, then another trigger is made for `sendTransaction` and those 2 triggers are caught in `ui/imports/shared/stores/DAppsStore.qml` ?

With the above flow I do not understand how the Popup is displayed on the Status desktop app.

**Review by kounkou — APPROVED** [review]
Just a curiosity question to understand how the different parts of the code are called.

**Review by alexjba — APPROVED** [review]
LGTM! Although I can't test it out to other issues.

**Pull request merged**